Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/core/PersonCertificatesExtensions.kt

```diff
@@ -80,38 +80,37 @@ private fun Collection<CwaCovidCertificate>.rule3findRecentRecovery(
     }.maxByOrNull { it.rawCertificate.recovery.validFrom }
 
 /**
- * 3
- * Series-completing Vaccination Certificate > 14 days:
- * Find Vaccination Certificates (i.e. DGC with v[0]) where v[0].dn equal to v[0].sd and the time difference
- * between the time represented by v[0].dt and the current device time is > 14 days, sorted descending by v[0].dt
- * (i.e. latest first).
+ * 4
+ * Vaccination Certificate <= 14 days
+ * Find Vaccination Certificates (i.e. DGC with v[0]) the time difference between the time represented by v[0].dt
+ * and the current device time is <= 14 days, sorted descending by v[0].dt (i.e. latest first).
  * If there is one or more certificates matching these requirements,
  * the first one is returned as a result of the operation.
  */
-private fun Collection<CwaCovidCertificate>.rule4FindRecentLastShot(
+private fun Collection<CwaCovidCertificate>.rule4findTooRecentFinalShot(
     nowUtc: Instant
 ): CwaCovidCertificate? = this
     .filterIsInstance<VaccinationCertificate>()
     .filter {
-        Days.daysBetween(it.rawCertificate.vaccination.vaccinatedOn, nowUtc.toLocalDateUtc()).days > 14
+        Days.daysBetween(it.rawCertificate.vaccination.vaccinatedOn, nowUtc.toLocalDateUtc()).days <= 14
     }
     .maxByOrNull { it.rawCertificate.vaccination.vaccinatedOn }
 
 /**
  * 5
- * Series-completing Vaccination Certificate <= 14 days
- * Find Vaccination Certificates (i.e. DGC with v[0]) where v[0].dn equal to v[0].sd and the time difference
- * between the time represented by v[0].dt and the current device time is <= 14 days,
- * sorted descending by v[0].dt (i.e. latest first).
+ * Vaccination Certificate > 14 days:
+ * Find Vaccination Certificates (i.e. DGC with v[0]) where the time difference between the time represented by v[0].dt
+ * and the current device time is > 14 days, sorted descending by v[0].dt
+ * (i.e. latest first).
  * If there is one or more certificates matching these requirements,
  * the first one is returned as a result of the operation.
  */
-private fun Collection<CwaCovidCertificate>.rule5findTooRecentFinalShot(
+private fun Collection<CwaCovidCertificate>.rule5FindRecentLastShot(
```

Comment:
did you switch two rules ? the git diff is really confusing

Reply:
Yes, the diff is confusing. I've changed rule4FindRecentLastShot -> rule5FindRecentLastShot and rule5FindRecentLastShot -> rule4findTooRecentFinalShot.